.github/workflows/generate-test-code.yml

```diff
@@ -1,6 +1,16 @@
 name: Generate test code
 on:
   workflow_dispatch:
+    inputs:
+      scope:
+        type: choice
+        description: scope of the test code to generate
+        required: true
+        default: autorest
+        options:
+          - all
+          - autorest
+          - cadl
```

Comment:
For now, default to "autorest".

I have yet to get "cadl" working. Script seems fine, but job will hang when invoking "java -jar".